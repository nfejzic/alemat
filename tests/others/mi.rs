use alemat::{elements::Ident, MathMl, MathMlFormatter};

#[test]
fn ident() {
    let out = MathMl::with_content(Ident::big_psi()).render(&mut MathMlFormatter);

    crate::snap_test!(out, name: "others_ident");
}
