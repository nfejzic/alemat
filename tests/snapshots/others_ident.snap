---
source: tests/others/mi.rs
expression: out
---
<math ><mi >Î¨</mi></math>
